mitodl/micromasters
# Deleted Jumbotron

#### What are the relevant tickets?

closes #1059 

#### What's this PR do?

This deletes the Jumbotron, a component we used to use on the dashboard and settings pages. We aren't using it any longer, so we might as well give it the axe.

#### Where should the reviewer start?

#### How should this be manually tested?

Just make sure that nothing broke? Not sure why it would, but always good to check.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### What GIF best describes this PR or how it makes you feel?

![](https://media.giphy.com/media/3oEduNF7DlpxgcHVJe/giphy.gif)
